packages/block-library/src/comment-template/index.php

```diff
@@ -31,6 +33,14 @@ function block_core_comment_template_render_comments( $comments, $block ) {
 
 		$children = $comment->get_children();
 
+		// We need to create the CSS classes BEFORE recursing into the children.
+		// This is because comment_class() uses globals like
+		// `$comment_alt` and `$comment_thread_alt` which are order-sensitive.
+		//
+		// The `false` parameter at the end means that we do NOT want the function to `echo` the output but to return a string.
+		// See https://developer.wordpress.org/reference/functions/comment_class/#parameters.
```

Comment:
Maybe we should use multi-comment here to match core standards - https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments.

Reply:
72d708f28a